### PR TITLE
Hamza's blogposts and project update

### PR DESCRIPTION
**Pull request by hamzaibrahim21**
### TODO

- [x] Add master thesis defense blogpost
- [x] Add lab joining as PhD blogpost
- [x] Add ECF-T paper blogpost
- [x] Update ECF-T VS pipeline to MolDockLab
- [x] Add MolDockLab figure
- [x] Add preprint paper
- [x] Add peer-reviewed paper

### Status

- [x] Ready to go

### CODE REVIEW

File: content/blog/hamza_member.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Hamza joins Volkamer lab"
+date: 2024-05-21T17:16:22+01:00
+author: hamza.ibrahim
+---
+
+Hamza holds a bachelor's degree in Pharmaceutical Sciences and a master's degree in Bioinformatics from Saarland University, and he is joining as a PhD candidate. He completed his master's thesis in the Volkamer Lab, developing MolDockLab.
```
Comment (PaulaKramer) [inline content/blog/hamza_member.md:7]: Can you link MolDockLab here?

**Review by PaulaKramer — COMMENTED** [review]
Thanks a lot @hamzaibrahim21, I just added one small comment.

**Comment by hamzaibrahim21** [comment]
> Thanks a lot @hamzaibrahim21, I just added one small comment.

Thanks for the feedback, I've linked it to MolDockLab in projects

**Review by AndreaVolkamer — CHANGES_REQUESTED** [review]
@hamzaibrahim21 thanks for hte blogpost., Can you check the Moldocklab links, they do not work for me?

**Comment by hamzaibrahim21** [comment]
> @hamzaibrahim21 thanks for hte blogpost., Can you check the Moldocklab links, they do not work for me?

@AndreaVolkamer Thanks for the feedback, it didn't work remotely. However, the links should work now

**Pull request merged**